zstacklib/zstacklib/test/utils/dump_debug_test.py

```
@@ -0,0 +1,132 @@
+from zstacklib.utils import debug
```

Ruff (0.8.2)

No such file or directory (os error 2)

(E902)